lib/services/routeparams.dart

```diff
@@ -32,7 +33,8 @@ class RouteParams {
   double distanceKm;
   List<String> poiCategories;
   AltitudeType altitudeType;
+  List<HikingRoute> routes;
```

Comment:
the initial idea was to not bundle the routes inside routeparams but i see no problem with it!